site/src/atomics/spacing.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Spacing Atomics
+description: Spacing related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Spacing Atomics
+
+Spacing Atomics can be used to add margin or padding on elements. They are especially helpful when composing layouts.
+
+## Pattern
```

Comment:
I think it might be good to start fairly fresh with this, here's structure I would find more intuitive, but we don't need to follow it exactly:

- class pattern (included link to **[this]https://github.com/microsoft/atlas-design/blob/main/css/src/atomics/README.md)**?
  - explanation of parts of class name
  - list of accepted cssproperties
  - list of accepted screensizes (this is something we'll want do on several of these documentation pages)
  - list of accepted value strings
- Usages
  - Adding uniform spacing
  - Adding spacing to a side
  - Adding spacing on a particular screen size
- Inventory
  - Full list of classes

Reply:
Something like this? https://design.docs.microsoft.com/pulls/186/atomics/spacing.html